config/gnu-warnings/cxx-9

```diff
@@ -0,0 +1,2 @@
+# Turn this on when the C++ wrappers obey the Rule of Five
+-Wno-deprecated-copy
```

Comment:
There are a LOT of these warnings, but adding things like move constructors seemed like a job for later.